mdm/api/src/main/java/com/gentics/mesh/core/data/dao/NodeDao.java

```diff
@@ -475,15 +481,4 @@
 	 * @return
 	 */
 	NodeReference transformToReference(HibNode node, InternalActionContext ac);
```

Comment:
The duplicate removed. As the function is called from the webroot context, I have left the `getWebrootEdges` name, and renamed Umberto's implementation.